en/04_Changelogs/5.0.0.md

```diff
@@ -26,6 +28,53 @@ title: 5.0.0 (unreleased)
+
+Some of those modules will provide CMS-5-compatible versions. Others will be dropped altogether.
+
+Email community@silverstripe.org ff you are keen to maintain some of the module that will no longer be officially supported.
+
+Module | Status | Notes
+-- | -- | --
+[bringyourownideas/silverstripe-composer-security-checker](https://packagist.org/packages/bringyourownideas/silverstripe-composer-security-checker) | Dropped | Underlying service does not work anymore.
+[composer/installers](https://github.com/composer/installers) | Dropped | Composer is more established than Silverstripe CMS.
+[cwp-themes/default](https://github.com/silverstripe/cwp-theme-default) | Dropped | 
+[cwp/cwp](https://packagist.org/packages/cwp/cwp) | CMS5 compatible without support | 
+[cwp/cwp-core](https://packagist.org/packages/cwp/cwp-core) | CMS5 comptabile without support | 
+[cwp/cwp-pdfexport](https://packagist.org/packages/cwp/cwp-pdfexport) | Dropped | 
+[cwp/cwp-search](https://packagist.org/packages/cwp/cwp-search) | CMS5 comptabile without support | 
+[hafriedlander/phockito](https://github.com/hafriedlander/phockito) | Dropped | 
+[hafriedlander/silverstripe-phockito](https://packagist.org/packages/hafriedlander/silverstripe-phockito) | Dropped | 
+[lekoala/silverstripe-debugbar](https://packagist.org/packages/lekoala/silverstripe-debugbar) | CMS5 comptabile without support | Debug bar is a development tool that should not be installed in production.
+[silverstripe/activedirectory](https://packagist.org/packages/silverstripe/activedirectory) | Dropped | 
+[silverstripe/akismet](https://packagist.org/packages/silverstripe/akismet) | Dropped | The parent library is outdated and they are better alternatives like [UndefinedOffset/silverstripe-nocaptcha](https://packagist.org/packages/undefinedoffset/silverstripe-nocaptcha)
+[silverstripe/behat-extension](https://github.com/silverstripe/silverstripe-behat-extension) | CMS5 comptabile without official support | 
+[silverstripe/ckan-registry](https://packagist.org/packages/silverstripe/ckan-registry) | CMS5 comptabile without support | 
+[silverstripe/comment-notifications](https://packagist.org/packages/silverstripe/comment-notifications) | CMS5 comptabile without official support | 
+[silverstripe/comments](https://packagist.org/packages/silverstripe/comments) | CMS5 comptabile without official support | 
+[silverstripe/content-widget](https://packagist.org/packages/silverstripe/content-widget) | CMS5 comptabile without support | 
+[silverstripe/fulltextsearch](https://packagist.org/packages/silverstripe/fulltextsearch) | CMS5 comptabile without support | 
+[silverstripe/graphql-devtools](https://packagist.org/packages/silverstripe/graphql-devtools) | CMS5 comptabile without official support | 
+[silverstripe/html5](https://packagist.org/packages/silverstripe/html5) | Folded back in core | 
```

Comment:
Should we be explicit whether core means Framework or CMS?

Reply:
That's a good idea. I'll do it.

Reply:
None of those modules were core modules. I've created a dedicated "supported module" page and link to it. The list of modules that has been split between core and optional modules.